style.css

```diff
@@ -433,308 +270,41 @@ button:hover, button:focus {
-}
-@media (min-width: 1100px) {
-	html { font-size: 98%; }
-}
-@media (min-width: 1200px) {
-	html { font-size: 100%; }
-}
-@media (min-width: 1300px) {
-	html { font-size: 102%; }
-}
-@media (min-width: 1400px) {
-	html { font-size: 104%; }
-}
-@media (min-width: 1500px) {
-	html { font-size: 106%; }
-}
-@media (min-width: 1600px) {
-	html { font-size: 108%; }
-}
-@media (min-width: 1700px) {
-	html { font-size: 110%; }
-}
-@media (min-width: 1800px) {
-	html { font-size: 112%; }
-}
-@media (min-width: 1900px) {
-	html { font-size: 114%; }
-}
-@media (min-width: 2000px) {
-	html { font-size: 116%; }
-}
-@media (min-width: 2100px) {
-	html { font-size: 118%; }
-}
-@media (min-width: 2200px) {
-	html { font-size: 120%; }
-}
-@media (min-width: 2300px) {
-	html { font-size: 122%; }
-}
-@media (min-width: 2400px) {
-	html { font-size: 124%; }
-}
-@media (min-width: 2500px) {
-	html { font-size: 126%; }
-}
-@media (min-width: 2600px) {
-	html { font-size: 128%; }
-}
-@media (min-width: 2700px) {
-	html { font-size: 130%; }
-}
-@media (min-width: 2800px) {
-	html { font-size: 132%; }
-}
-@media (min-width: 2900px) {
-	html { font-size: 134%; }
-}
-@media (min-width: 3000px) {
-	html { font-size: 136%; }
-}
-@media (min-width: 3100px) {
-	html { font-size: 138%; }
-}
-@media (min-width: 3200px) {
-	html { font-size: 140%; }
-}
-@media (min-width: 3300px) {
-	html { font-size: 142%; }
-}
-@media (min-width: 3400px) {
-	html { font-size: 144%; }
-}
-@media (min-width: 3500px) {
-	html { font-size: 146%; }
-}
-@media (min-width: 3600px) {
-	html { font-size: 148%; }
-}
-@media (min-width: 3700px) {
-	html { font-size: 150%; }
-}
-@media (min-width: 3800px) {
-	html { font-size: 152%; }
-}
-@media (min-width: 3900px) {
-	html { font-size: 154%; }
-}
-@media (min-width: 4000px) {
-	html { font-size: 156%; }
-}
-@media (min-width: 4100px) {
-	html { font-size: 158%; }
-}
-@media (min-width: 4200px) {
-	html { font-size: 160%; }
-}
-@media (min-width: 4300px) {
-	html { font-size: 162%; }
-}
-@media (min-width: 4400px) {
-	html { font-size: 164%; }
-}
-@media (min-width: 4500px) {
-	html { font-size: 166%; }
-}
-@media (min-width: 4600px) {
-	html { font-size: 168%; }
-}
-@media (min-width: 4700px) {
-	html { font-size: 170%; }
-}
-@media (min-width: 4800px) {
-	html { font-size: 172%; }
-}
-@media (min-width: 4900px) {
-	html { font-size: 174%; }
-}
-@media (min-width: 5000px) {
-	html { font-size: 176%; }
-}
-@media (min-width: 5100px) {
-	html { font-size: 178%; }
-}
 @media print {
 	@page {
 		margin: 2.75cm;
 	}
+	html {
+		padding: 0;
+	}
 	body {
 		color: #000;
 	}
-	body, th, td, .infobox, .warnbox, .button {
+	body,
+	th,
+	td,
+	.infobox,
+	.warnbox {
 		background: none;
 	}
-	table, th, td {
+	table,
+	th,
+	td {
 		border: 0.063em solid #000;
 		border-collapse: collapse;
 	}
 	a {
-		color: #5098CE;
 		text-decoration: underline;
 	}
 	a:after {
 		content: " (" attr(href) ")";
 		font-size: 80%;
 	}
-	#navigation, #footer {
+	.return-link {
```

Comment:
This rule should also apply to top-button (or the top button should share this class).

Reply:
Good catch. Fixed.